wgengine/magicsock/endpoint.go

```diff
@@ -99,6 +99,27 @@ type endpoint struct {
 	relayCapable    bool // whether the node is capable of speaking via a [tailscale.com/net/udprelay.Server]
 }
 
+// relayEndpointReady considers addr as de.bestAddr. It is only called by
```

Comment:
Mild phrasing suggestion: "relayEndpointReady considers addr as de.bestAddr" could be parsed as "unconditionally makes the given relay addr the best addr", depending on the definition of "consider" you use.

Maybe: "relayEndpointReady determines whether the given relay addr should be installed as de.bestAddr." ?

Reply:
updated in https://github.com/tailscale/tailscale/pull/16229/commits/03939df98c33ff4dfd9b713f3845c8c43aa9e49a